packages/data/src/index.js

```diff
@@ -7,7 +7,7 @@ import { combineReducers } from 'redux';
  * Internal dependencies
  */
 import defaultRegistry from './default-registry';
-export { loadAndPersist, withRehydration } from './persist';
+export { restrictPersistence, setPersistenceStorage } from './persist';
```

Comment:
This is breaking change. I think we should expose copies of `loadAndPersist` and `withRehydration` with deprecation warning informing about alternatives.